### DISCUSSION TITLE
adjust scoring ratio in osu!standard to 50% accuracy, 50% combo

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#14078] [Ideas]
the 50/50 split will emphasize that accuracy is at least as important as aiming consistently and will also reduce players getting "scorev1'ed"(better play overall, but overwritten by a high score play)

**Comment by gagahpangeran** [comment]
There has been such proposal for mania in the past and the result is changes of the ratio for mania to 99% accuracy and 1% combo, see https://github.com/ppy/osu/issues/12061

You can do something similar by gathering data from previous spotlight and ask suggestion and opinion from some player that played spotlight before about the ratio of scoring system.

_Maybe_ the core developer will consider such changes.

**Reply by smoogipoo** [reply]
Exactly. I’m very open to changing the scoring, especially since combo scoring in lazer is much harsher than sv1 or sv2, but giving just a ratio isn’t enough without supporting data.

**Reply by iSimpFor** [reply]
i'll see what can be done, i'll make a thread for this

**Reply by iSimpFor** [reply]
i opened a poll but got very few votes which is probably not enough of a sample size to use, i'll still put the poll results down:

results:
![results](https://user-images.githubusercontent.com/44514667/155926308-930de627-da11-4d68-9358-0e9a5a9bac3d.png)

**Reply by Moltanicaa** [reply]
I'll try to do something similar, but in a community with a coverage of ~ 1000 people. I'll add it in a couple of days.

**Reply by smoogipoo** [reply]
These polls aren't helpful. What I asked for was supporting data. Please refer to #12061 for an example of how things should be done.
